app/utils/scheduler.py

```
@@ -2419,17 +2435,28 @@ def _cleanup_pending_orders(self, send_slack_notification: bool = True):
-                        available_cash = float(cash_str)
+                present_balance_result = get_overseas_present_balance()
+                if present_balance_result.get("rt_cd") == "0":
+                    output3 = present_balance_result.get("output3", {})
+                    if output3:
+                        # frcr_use_psbl_amt: 외화사용가능금액 (USD)
+                        cash_str = output3.get("frcr_use_psbl_amt") or "0"
+                        try:
+                            available_cash = float(cash_str)
+                            logger.debug(f"[{function_name}] 💰 구매 가능 금액 (외화사용가능금액): ${available_cash:,.2f}")
+                        except (ValueError, TypeError) as e:
+                            logger.warning(f"[{function_name}] 외화사용가능금액 변환 실패: {cash_str}, 오류: {str(e)}")
```

Ruff (0.14.10)

Use explicit conversion flag

Replace with conversion flag

(RUF010)